src/meta/greedy_load_balancer.cpp

```diff
@@ -650,6 +650,101 @@ bool greedy_load_balancer::calc_disk_load(app_id id,
     }
 }
 
+struct flow_path
```

Comment:
Why would you define the struct and class in `.cpp` file, not in `.h`?

Reply:
Because other files don't need them